tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -865,6 +868,12 @@ bool FindFusedBatchNormEx(const RemapperContext& ctx, int node_index,
 
   // Input to a Relu can be an Add node with FusedBatchNorm as one of the inputs
   if (IsAdd(*relu_fanin_0_node_def)) {
+// Currently no CPU implementation for "FusedBatchNorm + SideInput +
+// <Activation>""
+#ifdef ENABLE_MKLDNN_V1
+    return false;
+#endif
```

Comment:
Nit: Can you align the indentation of the comment to match `return false;`? Or is this the formatter's doing?
```suggestion
    // Currently no CPU implementation for "FusedBatchNorm + SideInput +
    // <Activation>""
#ifdef ENABLE_MKLDNN_V1
    return false;
#endif
```

Reply:
Done